api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/UsMonthlySummaryCohortQueries.java

```diff
@@ -0,0 +1,131 @@
+  public CohortDefinition getActiveToEndDate() {
+    return generatlSql(
+        "activePatientsToEndDate", UsMonthlySummaryQueries.activePatientsToEndDate());
+  }
+
+  public CohortDefinition getEnrolledInReportingPeriod() {
+    return generatlSql(
+        "PACIENTES INSCRITOS NO PDF NUM PERIODO",
+        UsMonthlySummaryQueries.enrolledInReportingPeriod());
+  }
+
+  public CohortDefinition getReturnedToPDFInReportingPeriod() {
+    return generatlSql(
+        "PACIENTES QUE RETORNARAM AO PDF NUM PERIODO",
+        UsMonthlySummaryQueries.returnedToPDFInReportingPeriod());
+  }
+
+  /** @see UsMonthlySummaryQueries#inARTExcludingNotAnsweredAbandonedIn4WeeksUntilEndDate() */
+  public CohortDefinition getInARTExcludingNotAnsweredAbandonedIn4WeeksUntilEndDate() {
+    return generatlSql(
+        "inARTExcludingNotAnsweredAbandonedIn4WeeksUntilEndDate",
+        UsMonthlySummaryQueries.inARTExcludingNotAnsweredAbandonedIn4WeeksUntilEndDate());
+  }
+
+  /** B4PDF */
+  public CohortDefinition currentlyInARTWithoutPregnancy() {
+    String mappings = "startDate=${startDate},endDate=${endDate},location=${location}";
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
+    cd.setName("ACTUALMENTE EM TARV ATÃ‰ UM DETERMINADO PERIODO FINAL- PDF SEM INCLUIR GRAVIDAS");
+
+    CohortDefinition TARVACTUAL = getInARTExcludingNotAnsweredAbandonedIn4WeeksUntilEndDate();
+    CohortDefinition PDF = getActiveToEndDate();
+
+    cd.addSearch("TARVACTUAL", EptsReportUtils.map(TARVACTUAL, mappings));
+    cd.addSearch("PDF", EptsReportUtils.map(PDF, mappings));
+    cd.setCompositionString("TARVACTUAL AND PDF");
+    return cd;
+  }
+
+  public CohortDefinition getExitFromPdfTransferredTo() {
+    return generatlSql(
+        "PACIENTES QUE SAIRAM DO PDF - TRANSFERIDOS PARA",
+        UsMonthlySummaryQueries.exitFromPdfTransferredTo());
+  }
+
+  public CohortDefinition getComeOutOfPdfRemoved() {
+    return generatlSql(
+        "PACIENTES QUE SAIRAM DO PDF - RETIRADOS", UsMonthlySummaryQueries.comeOutOfPdfRemoved());
+  }
+
+  public CohortDefinition getComeOutOfPdfObitos() {
+    return generatlSql(
+        "PACIENTES QUE SAIRAM DO PDF - OBITOS", UsMonthlySummaryQueries.comeOutOfPdfObitos());
+  }
+
+  public CohortDefinition getComeOutOfPdfSuspension() {
+    return generatlSql(
+        "PACIENTES QUE SAIRAM DO PDF - SUSPENSO", UsMonthlySummaryQueries.comeOutOfPdfSuspension());
+  }
+
+  public CohortDefinition getClinicalConsultationWithinReportingPeriod() {
+    return generatlSql(
+        "PACIENTES PDF QUE TIVERAM CONSULTA CLINICA NUM PERIODO",
+        UsMonthlySummaryQueries.clinicalConsultationWithinReportingPeriod());
+  }
+
+  private CohortDefinition generatlSql(String name, String query) {
+    CohortDefinition cd = genericCohortQueries.generalSql(name, query);
+    addParameters(cd);
+    return cd;
+  }
+
+  private void addParameters(CohortDefinition cd) {
```

Comment:
Already those parameters are supplied by **genericCohortQueries.generalSql** and passed already for every cohort that use that method, do we need to replicate it here? or we can just reuse those